### ISSUE TITLE
Request for new ontology [PROMOT]

### ISSUE THREAD

**Issue by JuPoIm** [OBOFoundry/OBOFoundry.github.io#2943]
### Title

PROMOT Ontology

### Short Description

PROMOT Ontology describes three groups of neuromuscular diseases, their related gene variants, phenotypes, and affected anatomical entities, body structures, body functions and activities.

### Description

The PROMOT ontology represents three groups of neuromuscular diseases related to the 13 genes the PROMOT Master Observational Trial focuses on:

- Oculopharyngeal Muscular Dystrophy (_PABPN1_)
- Congenital Myasthenic Syndromes (_CHRNE_, _DOK7_, _COLQ_, _RAPSN_, _GFPT1_, _GMPPB_)
- Congenital Myopathies (_RYR1_, _TTN_, _NEB_, _MTM1_, _ACTA1_, _SELENON_)

This ontology describes the phenotypes associated to the diseases and genes. It also describes the affected anatomical entities, body structures, body functions, and activities. 

Intended users are the experts from the PROMOT project but also anyone interested in neuromuscular diseases and their impacts on functioning and activities, such as the newly created NMDO project consortium.

The ontology relates to the following OBO ontologies: BFO, ECO, HP, IAO, NCIT, OBI, RO and SO.

The PROMOT ontology was built in the context of the "Performing a Rare Disease-Oriented Master Observational Trial" initiative (PROMOT initiative). It aims to be the conceptual schema for the PROMOT initiative common data model. This common data model is intended to collect and share in a federated infrastructure patient data scattered between the different sites involved in the master observational trial.

This ontology is also available on [Bioportal ](https://bioportal.bioontology.org/ontologies/PROMOT) under the same name.

### Identifier Space

PROMOT

### License

CC-BY 4.0

### Domain

health

### Source Code Repository

https://github.com/JuPoIm/promot/

### Homepage

https://github.com/JuPoIm/promot/

### Issue Tracker

https://github.com/JuPoIm/promot/issues

### Contribution Guidelines

https://github.com/JuPoIm/promot/blob/master/CONTRIBUTING.md

### Ontology Download Link

https://github.com/JuPoIm/promot/blob/master/promot.owl

### Contact Name

Juliette POTIER

### Contact Email

juliette.potier@institutimagine.org

### Contact GitHub Username

JuPoIm

### Contact ORCID Identifier

0009-0000-5160-7927

### Formats

- [x] OWL RDF/XML (.owl)
- [x] OBO (.obo)
- [x] OBO Graph JSON (.json)

### Dependencies

- bfo
- eco
- hp
- iao
- ncit
- obi
- ro
- so

### Usages

```yaml
- user: https://www.promotinitiative.com/
  description: PROMOT initiative
```

### Intended Use Cases and/or Related Projects

Intended use case is the "Performing a Rare Disease-Oriented Master Observational Trial" initiative (PROMOT initiative). The PROMOT ontology aims to be the conceptual schema for the common data model the PROMOT initiative will use to collect patient data scattered between the different sites involved in the master observational trial.

More about PROMOT initiative can be found at [https://www.promotinitiative.com/project](https://www.promotinitiative.com/project).

Related project is the newly created NMDO project consortium that wishes to include PROMOT (https://github.com/NeuromuscularDisease/neuromuscular-disease-ontology)

### Data Sources

For OPMD:

1) Scoping Reviews:
- Bélair N, Gagnon C, Duchesne E. What is known about muscle weakness, balance impairments and indoor mobility limitations in oculopharyngeal muscular dystrophy? A scoping review. J Neuromuscul Dis. 2025 Dec 30:22143602251397052. doi: 10.1177/22143602251397052. Epub ahead of print. PMID: 41467981.
- Côté C, Gagnon C, Youssof S, et al. The requirement for a disease-specific patient-reported outcome measure of dysphagia in oculopharyngeal muscular dystrophy. Muscle Nerve. 2019;59:445–50. doi: 10.1002/mus.26405

2) Research protocol from "Performing a Rare Disease-Oriented Master Observational Trial" initiative, European Joint Programme on Rare Diseases, https://www.promotinitiative.com/
- Clinical Registry Form
- Clinical Research Form

3) HPO KB associations, Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, et al. The Human Phenotype Ontology in 2024: phenotypes around the world. Nucleic Acids Res. 2024 Jan 5;52(D1):D1333-D1346. doi: 10.1093/nar/gkad1005. PMID: 37953324; PMCID: PMC10767975.

4) Consensus meetings with disease experts

For CMYO:

1) Scoping Review from "Performing a Rare Disease-Oriented Master Observational Trial" initiative, European Joint Programme on Rare Diseases, https://www.promotinitiative.com/ (submitted for publication)

2) HPO KB associations, Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, et al. The Human Phenotype Ontology in 2024: phenotypes around the world. Nucleic Acids Res. 2024 Jan 5;52(D1):D1333-D1346. doi: 10.1093/nar/gkad1005. PMID: 37953324; PMCID: PMC10767975.

4) Consensus meetings with disease experts

For CMS:

1) Scoping Review from "Performing a Rare Disease-Oriented Master Observational Trial" initiative, European Joint Programme on Rare Diseases, https://www.promotinitiative.com/ (submitted for publication)

2) HPO KB associations, Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, et al. The Human Phenotype Ontology in 2024: phenotypes around the world. Nucleic Acids Res. 2024 Jan 5;52(D1):D1333-D1346. doi: 10.1093/nar/gkad1005. PMID: 37953324; PMCID: PMC10767975.

4) Consensus meetings with disease experts

### Additional comments or remarks

The PROMOT ontology is released in english (promot-en.owl) as well as in three languages (promot-int.owl) which are English, French and Spanish because the common data model should handle the languages spoken in the different sites of the master observational trial.

The international version is currently hosted on [Bioportal](https://bioportal.bioontology.org/ontologies/PROMOT) under the same name.

### OBO Foundry Pre-registration Checklist

- [x] I have read and understood [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md).
- [x] There is no other ontology in the OBO Foundry which would be an appropriate place for my terms. If there were, I have contacted the editors, and we decided in mutual agreement that a separate ontology is more appropriate.
- [x] My ontology has a specific release file with a version IRI and a `dc:license` annotation, serialised in RDF/XML.
- [x] My identifiers (classes and properties IRIs) are formatted according to the [OBO Foundry Identifier Policy](http://obofoundry.org/id-policy)
- [x] My term labels are in English and conform to the OBO Foundry [Naming Conventions](http://obofoundry.org/principles/fp-012-naming-conventions.html)
- [x] I understand that term definitions are key to understanding the intentions of a term, especially when the ontology is used in curation. I made sure that a reasonable majority of terms in my ontology--and all top level terms--have definitions, in English, using the [IAO:0000115](https://ontobee.org/ontology/IAO?iri=http://purl.obolibrary.org/obo/IAO_0000115) property.
- [x] For every term in my ontology, I checked whether another OBO Foundry ontology has one with the same meaning. If so, I re-used that term directly (not by cross-reference, by directly using the IRI).
- [x] For all relationship properties (Object and Data Property), I checked whether [the Relation Ontology (RO)](https://ontobee.org/ontology/catalog/RO?iri=http://www.w3.org/2002/07/owl%23ObjectProperty) includes an appropriate one. I understand that aligning with RO is an essential part of the overall alignment between OBO ontologies!
- [x] For the selection of appropriate annotation properties, I looked at [OMO](https://ontobee.org/ontology/catalog/OMO?iri=http://www.w3.org/2002/07/owl%23AnnotationProperty) first. I understand that aligning ontology metadata and term-level metadata is essential for cross-integration of OBO ontologies.
- [x] If I was not sure about the meaning of any of the checkboxes above, I have consulted with a member of the OBO Foundry for advice, e.g., through the [obo-discuss](https://groups.google.com/g/obo-discuss) Google Group.
- [x] The requested ID space does not conflict with another ID space found in other registries such as the Bioregistry and BioPortal, see [here](https://obofoundry.org/id-policy) for a complete list.

**Comment by pfabry** [comment]
Dear @JuPoIm, 

Thank you for your submission. The review will be executed as a two stage process:

First, you will have to pass [OBO NOR Dashboard](https://obofoundry.org/obo-nor.github.io/dashboard/index.html). Pass means that no check apart from Users and Versioning may be red. In addition, a new check is currently implemented: it consists in a lexical match of your original terms with those already existing in OBO Foundry published ontologies. The goal is to prevent the duplication of terms with similar meanings (cf. [Review SOP](http://obofoundry.org/docs/SOP.html#ROOM)). A list of terms that are potential duplicates will be provided soon.

After you have successfully passed the Dashboard you will be assigned an OBO Operations committee member to review the ontology. The assigned reviewer is to be considered the final arbiter of requirements; look to that reviewer's guidance regarding which suggestions made by other reviewers must be done, which suggestions are simply good to do but not required, and which should not be done.

Usually, the review will result in an opportunity for you to improve the ontology. When the reviewer believes the ontology is ready for presentation to the OBO Operations Committee, they will present your ontology during an OBO Operations Call. This gives other members of the committee the opportunity to assess your work.

When a decision is reached by the committee you will be informed here on the issue tracker. The process can take any number of weeks or months, depending on the case at hand.
Please let us know about any reasons you might have for increased urgency.

You will be informed once your ontology is loaded in the OBO NOR Dashboard.

Good luck!

**Comment by pfabry** [comment]
The result of the lexical matching is as follows:

Analysis of PROMOT run on Thu Aug 20 15:48:23 2026 


## Failed Nodes (14/197; 7.1%)

| PROMOT   | name                 | obo-curie                                              | obo-name                       |   obo-score |
|----------|----------------------|--------------------------------------------------------|--------------------------------|-------------|
| 0010015  | Adulthood            | [`ecao:0000530`](https://semantic.farm/ecao:0000530)   | adulthood                      |       0.762 |
|          |                      | [`gsso:003825`](https://semantic.farm/gsso:003825)     | adulthood                      |       0.762 |
| 0010017  | Middle adulthood     | [`gsso:006872`](https://semantic.farm/gsso:006872)     | middle adulthood               |       0.772 |
| 0010018  | Young adulthood      | [`gsso:006873`](https://semantic.farm/gsso:006873)     | young adulthood                |       0.772 |
| 0010028  | Congenital           | [`mondo:0021140`](https://semantic.farm/mondo:0021140) | congenital                     |       0.762 |
| 0010031  | Childhood            | [`gsso:003826`](https://semantic.farm/gsso:003826)     | childhood                      |       0.762 |
| 0010041  | Consent status       | [`opmi:0000350`](https://semantic.farm/opmi:0000350)   | consent status                 |       0.772 |
|          |                      | [`scdo:1000490`](https://semantic.farm/scdo:1000490)   | Consent Status                 |       0.772 |
| 0010044  | Active               | [`pato:0002354`](https://semantic.farm/pato:0002354)   | active                         |       0.762 |
| 0010045  | Rejected             | [`cio:0000039`](https://semantic.farm/cio:0000039)     | rejected                       |       0.762 |
| 0010046  | Inactive             | [`pato:0002355`](https://semantic.farm/pato:0002355)   | inactive                       |       0.762 |
| 0110001  | Activity             | [`xco:0000001`](https://semantic.farm/xco:0000001)     | activity                       |       0.762 |
| 2000001  | affects              | [`ro:0002264`](https://semantic.farm/ro:0002264)       | acts upstream of or within     |       0.556 |
|          |                      | [`ro:0002418`](https://semantic.farm/ro:0002418)       | causally upstream of or within |       0.556 |
| 2000003  | has genetic mutation | [`cido:0000393`](https://semantic.farm/cido:0000393)   | has genetic mutation           |       0.778 |
| 2000004  | has identifier       | [`opmi:0000481`](https://semantic.farm/opmi:0000481)   | has identifier                 |       0.778 |
| 2000009  | measures             | [`gsso:010944`](https://semantic.farm/gsso:010944)     | measure                        |       0.778 |
|          |                      | [`scdo:1000194`](https://semantic.farm/scdo:1000194)   | measures                       |       0.778 |

Cf. attached file.

[PROMOT lexmatch 20260820.md](https://github.com/user-attachments/files/31279342/PROMOT.lexmatch.20260820.md)

**Comment by JuPoIm** [comment]
Dear Paul,

Thank you for your feedback. I have edited the ontology to resolve the URIs, definitions, naming and report issues in the Obo Dashboard.
(Note that I have edited the Ontology Download Link in the submission form to fit the new file name: promot.owl corresponding now to the formerly promot-en.owl).

Is it impossible to submit for a https://creativecommons.org/licenses/by-nc-sa/4.0/ licence with OboFoundry? This is, for now, the licence selected by the legal partner of PROMOT.

Thank you for the lexical matching list, I will discuss it with the rest of the team to see which terms we can include in PROMOT (at first sight, SCDO terms look nice, same for XCO activity, the thing for adulthood and co is that we wanted to keep our definitions similar to HP Onset's terms but we will consider the suggestions).

Many thanks!

**Comment by matentzn** [comment]
> Is it impossible to submit for a [creativecommons.org/licenses/by-nc-sa/4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) licence with OboFoundry? This is, for now, the licence selected by the legal partner of PROMOT.

https://obofoundry.org/principles/fp-001-open.html

```
OBO Foundry Ontologies MUST EITHER be released under a Creative Commons Attribution 3.0 Unported (CC BY 3.0) license or later (e.g. Attribution 4.0 International (CC BY 4.0), 
OR released into the public domain under [Creative Commons CC0 1.0 Public Domain Dedication (CC0 1.0)](https://creativecommons.org/publicdomain/zero/1.0/).
```

**Comment by pfabry** [comment]
> > Is it impossible to submit for a [creativecommons.org/licenses/by-nc-sa/4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) licence with OboFoundry? This is, for now, the licence selected by the legal partner of PROMOT.
> 
> https://obofoundry.org/principles/fp-001-open.html
> 
> ```
> OBO Foundry Ontologies MUST EITHER be released under a Creative Commons Attribution 3.0 Unported (CC BY 3.0) license or later (e.g. Attribution 4.0 International (CC BY 4.0), 
> OR released into the public domain under [Creative Commons CC0 1.0 Public Domain Dedication (CC0 1.0)](https://creativecommons.org/publicdomain/zero/1.0/).
> ```
As CC BY-NC-SA 4.0 allows for modification of the ontology (my understanding is that the difference with CC BY 4.0 is that it prohibits commercial reuse) should it fall under pricnple 1 ?

**Comment by balhoff** [comment]
Prohibiting commercial use is against principle 1.

**Comment by JuPoIm** [comment]
Hello everyone,

Thank you for all the informations you provided to improve the ontology!

Following discussion, we see no issue by publishing the ontology under the CC BY 4.0 licence. PROMOT Ontology is now available under this licence, following principle 1.


Regarding the lexical matching, we have included the following suggestions:

- PROMOT:2000009 has been replaced by SCDO:1000194 'measures'
- PROMOT:2000004 has been replaced by OPMI:0000481 'has identifier'
- PROMOT:0110001 has been replaced by XCO:0000001 'activity'
- PROMOT:0010041 has been replaced by SCDO:1000490 'Consent Status'

Also:
- PROMOT:2000003 has been replaced by [VO:0001039](https://www.ebi.ac.uk/ols4/ontologies/vo/properties/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FVO_0001039) 'has gene mutation' as it provides a definition unlike [CIDO:0000393](https://semantic.farm/cido:0000393)

However, the hereafter suggestions do not have the targeted meaning for PROMOT:

PROMOT:0010044 | Active | [pato:0002354](https://semantic.farm/pato:0002354) | active | 0.762 -> definition is about a physical object
PROMOT:0010045 | Rejected | [cio:0000039](https://semantic.farm/cio:0000039) | rejected | 0.762 -> does not seem to apply to a consent status
PROMOT:0010046 | Inactive | [pato:0002355](https://semantic.farm/pato:0002355) | inactive | 0.762 -> definition is about a physical object
PROMOT:2000001 | affects | [ro:0002264](https://semantic.farm/ro:0002264) | acts upstream of or within | 0.556
PROMOT:2000001 | affects | [ro:0002418](https://semantic.farm/ro:0002418) | causally upstream of or within | 0.556 -> we would like to keep the definition quite broad

Regarding the suggested 'age group' related entities, they are close to what we wish to have in PROMOT but we would like to keep definitions close to the ones provided by the descendants of [HP:0003674 'Onset'](https://hpo.jax.org/browse/term/HP:0003674) as we include them in PROMOT. Changing some of the PROMOT subcategories will go against the definitions of the remaining terms (e.g. GSSO childhood : The time during which one is a child, from between infancy and puberty vs HPO Childhood onset : Onset of disease at the age of between 1 and 5 years). I hope this is ok to keep the PROMOT definitions.

Otherwise, Licence, URIs, Definitions, Naming and ROBOT Report issues in the Obo Dashboard should be now resolved.

I would be happy to include any further needed changes.

Kind regards

**Comment by pfabry** [comment]
Dear @JuPoIm,

Your ontology has passed the dashboard. A reviewer will be assigned shortly.
